storage/database.go

```diff
@@ -304,7 +348,7 @@ func (d *db) splayedTick() {
 	}
 }
 
-func (d *db) nextUniqueIndex() uint64 {
+func (d *db) next() uint64 {
```

Comment:
We already have `index` in the interface name `increasingIndex`, so figured `next` is self- explanatory enough. Can change it to `nextIndex` if that's more clear.